addon/components/frost-chart-y-axis.js

```diff
@@ -118,15 +119,21 @@ export default Component.extend({
 
   init () {
     this._super(...arguments)
-    this.attrs.dispatch({
+    this.dispatch({
       type: 'REGISTER_AXIS'
     })
   },
 
-  didInsertElement () {
-    this._super(...arguments)
-    run.scheduleOnce('afterRender', this, this._dispatchRenderedAxis)
-  }
-
   // == Actions ===============================================================
+  actions: {
+    _dispatchTickRendered (action) {
```

Comment:
It's already easy to distinguish though via Ember conventions/usage.  You can/do not call actions from outside of Ember components, nor is there really a way to do so.  Only actions can be called via the "actiony" way in Ember so knowing an action is being called and that it's an action because it's an underscore is superfluous structure.